NitroxClient/MonoBehaviours/Gui/Input/KeyBindings/Actions/ChatKeyBindingAction.cs

```diff
@@ -7,11 +7,7 @@ public class ChatKeyBindingAction : KeyBindingAction
     {
         public override void Execute()
         {
-            // If no other UWE input field is currently active then allow chat to open.
-            if (FPSInputModule.current.lastGroup == null)
```

Comment:
Does it work as attended when writing on a base sign for example ?

Reply:
Hm. Good point, I'll test on a different save, gimme a few minutes.

Reply:
Okay it'll take some time until I'm able to test it, my more advanced save is on the legacy branch and the master has only a save at the starting point :D

Reply:
You know that you can just cheat what you need for testing? Even spawn in creative if set up in the server.

Reply:
`/changeservergamemode creative` inside chat or `changeservergamemode creative`` console if you don't know how to use IG command. or you can edit server config file accordingly as Jannify said

Reply:
Ah, yeah I found out there was creative mode available while I was transferring save, then just had difficulties figuring out how to switch to creative on the fly. :D

But yeah, it seems it will also override sign text input, so no good like this. Hmm.